docs/how-to/mgc-setup.md

```diff
@@ -0,0 +1,163 @@
+# Setting up MGC in Existing OCM Clusters
+
+This guide will show you how to install and configure the Multi-Cluster Gateway Controller in preexisting [Open Cluster Management](https://open-cluster-management.io/)-configured clusters.
+
+## Prerequisites
+
+- A **hub cluster** running the OCM control plane (v0.11.0 or greater)
+- Any number of additional **spoke clusters** that have been configured as OCM [ManagedClusters](https://open-cluster-management.io/concepts/managedcluster/)
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) (>= v1.14.0)
+- [Kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/)
+- [Helm](https://helm.sh/docs/intro/quickstart/#install-helm)
```

Comment:
I would consider swapping the Helm with a cert-manager as part of the prerequisites.

Reply:
It will be if you are installing cert-manager. If you are just installing the controller, probably not.

Reply:
Helm is required if installing `cert-manager`.

Reply:
Apologies @jsmolar - I only saw your first message when I replied earlier. I'll add something in the prerequisites to the effect of `either a preexisting cert-manager installation or the kustomize and helm CLIs so that it can be installed`.

Reply:
@eoinfennessy no worries. Thank you! My suggestion was based on a discussion we had earlier, that we should avoid using more tools than necessary in one example/quick start (Kustomize and Helm)